cli/tests/unit_node/_fs/_fs_open_test.ts

```diff
@@ -0,0 +1,401 @@
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+import {
+  O_APPEND,
+  O_CREAT,
+  O_EXCL,
+  O_RDONLY,
+  O_RDWR,
+  O_SYNC,
+  O_TRUNC,
+  O_WRONLY,
+} from "./_fs_constants.ts";
+import {
+  assert,
+  assertEquals,
+  assertThrows,
+  fail,
+} from "../../../../test_util/std/testing/asserts.ts";
+import { assertCallbackErrorUncaught } from "../_test_utils.ts";
+import { open, openSync } from "node:fs";
+import { join, parse } from "node:path";
+import { existsSync } from "node:fs";
+import { closeSync } from "node:fs";
```

Comment:
nit: Can you make these imports from "node:fs" into single statement? (like `import { existsSync, closeSync } from "node:fs";`